Update npm request module

rhernand3z opened the pull request:
This PR updates all NPM modules to newer stable releases. Mainly to fix a bug from the current request module version this module was using that threw a weird POST invalid URI error that has been fixed on a newer version.

By running npm run test, the latest Standard JS spec expected `new Error()` when rejecting a promise. See: https://eslint.org/docs/rules/prefer-promise-reject-errors

Tested and linted all code. 👍

nodanaonlyzuul commented:
Great work @rhernand3z!
Thanks for doing this...I'm going to:

* merge this
* publish it to npm
* push a "1.0.4" tag to github

Can you link me to the issue in `request` module just so I can read up on it?

rhernand3z commented:
Hi Buddy,

I added the issue URL in the CHANGELOG for this version bump:

- Updated NPM packages to address npm request module bug https://github.com/request/request/issues/1595

nodanaonlyzuul commented:
YOU ARE A HERO
